src/frontend/src/handler/create_sink.rs

```diff
@@ -371,6 +371,9 @@ pub async fn get_partition_compute_info(
 async fn get_partition_compute_info_for_iceberg(
     iceberg_config: &IcebergConfig,
 ) -> Result<Option<PartitionComputeInfo>> {
+    // TODO: enable partition compute for iceberg after fixing the issue of sink decoupling.
```

Comment:
Is this a temporary fix ? Do we need a configurable switch?

Reply:
Yes, it is a temporary fix, and some users are waiting for it. I think we don't need a switch because it can't work at all right now 🥵 .